tests/check/check_h26xsigned_auth.c

```diff
@@ -1713,34 +1713,34 @@ static void validation_helper_function(signed_video_t *sv, nalu_list_item_t *sei
   }
 }
 
-typedef struct pk_setting {
-  bool pk_in_sei;
-  bool wrong_pk;
-  bool signature_info;
-  bool set_public_key;
-} pk_setting;
-
-pk_setting pk_tests[6] = {
-  // Public key on validation side from start
-  {false, false, false, true},
-  // Public key in sei and on validation side from start
-  {true, false, false, true},
-  // No public key
-  {false, false, false, false},
-  // Public key on validation side later
-  {false, false, true, false},
-  // Public key in sei and on validation side later
-  {true, false, true, false},
-  // Public key in sei and bad public key on validation_side
-  {true, true, true, false},
-};
-
 /* Test description */
 START_TEST(public_key_on_validation_side_from_start)
 {
   // This test runs in a loop with loop index _i, corresponding to struct sv_setting _i in
   // |settings|; See signed_video_helpers.h.
 
+  struct pk_setting {
+    bool pk_in_sei;
+    bool wrong_pk;
+    bool use_signature_info;
+    bool set_pk;
+  };
+
+  struct pk_setting pk_tests[6] = {
+    // Public key on validation side from start
+    {false, false, false, true},
+    // Public key in sei and on validation side from start
+    {true, false, false, true},
+    // No public key
+    {false, false, false, false},
+    // Public key on validation side later
+    {false, false, true, false},
+    // Public key in sei and on validation side later
+    {true, false, true, false},
+    // Public key in sei and bad public key on validation_side
+    {true, true, true, false},
+  };
+
   for (int j = 0; j < 6; j++) {
```

Comment:
You should preferably avoid magic number `6` here. Replace with `sizeof(pk_tests) / sizeof(*pk_tests)` or define a constant.